comp230_01_1607539.tex

```diff
@@ -0,0 +1,65 @@
+Alternatively, Linden Labs (\textit{Second Life}) have embraced the idea of player-generated intellectual property. President and CEO Phillip Rosedale said ``We allow people to create a world which will be thousands of times more compelling than we could create ourselves'' \cite{herman2006your}. Now, the game world is rife with bootlegged videos and audio streams of popular music \cite{herman2006your}. Whilst \textit{Wikipedia} uses scripts that perform the automated tasks of policing content and correcting vandalism \cite{hong2014becoming}, censorship imposes a cost that must be part of a consequentialist moral calculation \cite{schulzke2010defending}.
+
+Modders are an evolution of the consumer, reconfiguring the position of the user. No longer simply purchasing the game as a product, but being actively engaged in its continued development and marketing \cite{herman2006your}. With \textit{Skyrim}, the modders also constituted a source of free work upon which Bethesda could capitalize \cite{hong2013game}. If the average large mod requires more than 1,000 work hours to complete, we can estimate the development cost saved by referencing salaries for the profession \cite{postigo2007mods}. Modders unpaid labour can make their work prone to industry appropriation, but can also serve as part of a portfolio \cite{nieborg2008mod}. id Software, the company which produced \textit{DOOM}, discovered many of their current employees and development partners based on mods that were created and distributed over the Internet \cite{hong2014becoming}. 
+
+\subsection*{Conclusion}
+
+Inventive mods can not only strengthen the unit sales of the games from which they originated; their innovative playing format can author entirely new genres of games \cite{hong2014becoming}, and offer recruitment opportunities.
+
+However, the risks in allowing mods include not just bringing the brand of the intellectual property (IP) into disrepute, but also tarnishing the reputation of the developers and any future work they may accomplish. Considering the public image of a game's brand includes questioning if it is ethical for this game to exist? Should this game have been created in the first place? What in-game actions are defined as ``good'' by the game \cite{zagal2009ethically}? A mod, either as an extension or total conversion, should be subject to the same rigor. Serious attention must be given to the gap that may exist between modders' design desires and corporate interest in the given models of property and authorship \cite{taylor2002whose}. Takedown notices can be served, but community backlash can adversely affect profits and goodwill.
```

Comment:
Consider merging these two paragraphs.

Reply:
I am left wondering if you are advocating the nurturing of a modding community, and advising readers that they have legal remedies should they encounter one the risks that you've outlined. A stronger statement to this---or the intended---effect would make for a stronger essay. Declare your position, make it clear. This will make for a better and more convincing essay.